### PR TITLE
Update files for the release

### PR DESCRIPTION
**Pull request by fabienrohrer**
- [x] Update translation files
- [x] Update release date in the change log

### CODE REVIEW

**Comment by fabienrohrer** [comment]
test me

**Comment by fabienrohrer** [comment]
👍

File: change_logs/ChangeLog.html [hunk 1]
```diff
@@ -10,7 +10,7 @@
 <h1>Webots change log</h1>
 
 <h2>Webots R2019a revision 1</h2>
-Released on XXX XXth, 2019
+Released on January 7th, 2019
```
Comment (omichel) [inline change_logs/ChangeLog.html]: February

Comment (fabienrohrer) [reply change_logs/ChangeLog.html]: haha sorry

**Comment by fabienrohrer** [comment]
test me

**Pull request merged**